src/main/webapp/portal/features/partials/features.html

```diff
@@ -0,0 +1,31 @@
+<div ng-controller="FeaturesController as featuresCtrl" class="row portlet-frame">
+  <div class="portlet-header">
+    <img src="img/square.jpg" alt="UW Campus cover image">
+    <h1>{{NAMES.title}} New Features</h1>
+    <p class="portlet-description">Stay up-to-date on the newest features of {{NAMES.title}}.</p>
+  </div>
+
+  <div class="row">
+    <div class="col-xs-12">
+      <div class="features-feed">
+				<div class="feature-item" ng-repeat="feature in features | orderBy: '-id'">
+	        <h5 class="date">{{feature.goLiveMonth + 1}}/{{feature.goLiveDay}}/{{ feature.goLiveYear }}</h5>
+	        <h3>{{ feature.title }}</h3>
+	        <p>{{ feature.description }}</p>
+					<p ng-if="feature.learnMoreURL">Learn more at <a ng-href="{{ feature.learnMoreURL }}">{{ feature.learnMoreURL }}</a>.</p>
+	        <div class="image-div">
+						<img ng-if="feature.img" ng-src="{{feature.img}}">
+					</div>
+				</div>
+      </div>
+    </div>
+		<br><br><br><br><br>
```

Comment:
Yea I suppose that's bad practice. I'm not a huge fan of one-off margin styles either, but it's probably better than <br>s :P